### ISSUE TITLE
Hotkey CMD+§ doesn't work for quake style

### ISSUE THREAD

**Issue by ideea** [warpdotdev/Warp#856]
### Describe the bug

```text
CMD+§ doesn't work as shortcut.
```

### Screenshots

<img width="513" alt="image" src="https://user-images.githubusercontent.com/318955/160988939-edb91ec2-1f14-4f13-a0a9-357a6b42f9c1.png">

### Operating System

MacOS

### OS Version

12.3

**Comment by jovobe** [comment]
The hotkey ⌘+<+⇧ doesn't work either. I have added Warp under Security & Privacy, checked it there and restarted it. I am on v0.2022.03.29.02.23.stable_01 on macOS 12.3.

**Comment by kevinyang372** [comment]
> The hotkey ⌘+<+⇧ doesn't work either. I have added Warp under Security & Privacy, checked it there and restarted it. I am on v0.2022.03.29.02.23.stable_01 on macOS 12.3.

Hi @jovobe! Thanks for reporting this. Will push for a fix next week!

**Comment by kevinyang372** [comment]
For `CMD+§`, unfortunately this is a deeper bug in our UI framework for handling alt modified keys, I have created a ticket internally to keep track of this and will report back once this is fixed!

**Comment by jovobe** [comment]
@kevinyang372 I just updated Warp to v0.2022.04.01.01.37.stable_02 and ⌘+<+⇧ is still not working.

**Comment by kevinyang372** [comment]
@jovobe Just to make sure, is the keybinding `CMD+SHIFT+UP`? It works for me on the latest version of warp. Does the keybinding show up when you register it in the settings editor? If it doesn't show up at all, it could be because some other apps is using this global hotkey or this is a system hotkey.

**Comment by jovobe** [comment]
@kevinyang372 no, it is `CMD+SHIFT+<`. The last key is the "less-than sign". I just tested it again on version v0.2022.04.01.01.37.stable_03, and it is still not working. The keybinding does not show up, when I try to register it. It is not possible that other apps are using this key combination, because iTerm2 and BetterTouchTool can register to it. Both are disabled, when I try to register the hotkey in Warp. But when I try to register it in one of the other apps, it just works.
This is from BetterTouchTool: ![Screenshot 2022-04-05 at 21 51 16](https://user-images.githubusercontent.com/1265894/161838070-b7344a22-2289-4a3d-838d-be56e5850dd0.png)
And this is from iTerm2: ![Screenshot 2022-04-05 at 21 52 14](https://user-images.githubusercontent.com/1265894/161838081-d644bb52-cf94-4224-bc04-4f6dba65c70c.png)

**Comment by kevinyang372** [comment]
@jovobe Ah I see. Sorry about this! I misread the keybinding before -- yes this is a known issue and it has the same underlying problem with CMD+§.

TLDR on the issue in our UI framework: we receive keybindings with their produced character rather than the keycode. But for registering global hotkey, the Carbon API requires the keycode for the keys. This discrepancy unfortunately causes some keys to not be registered properly.

**Comment by ChristianVermeulen** [comment]
@kevinyang372 Is it possible to give an estimation for when it might be fixed? It's pretty much the only thing stopping me from going full Warp at this point :D

**Comment by ChristianVermeulen** [comment]
I just found this to be duplicate of https://github.com/warpdotdev/Warp/issues/966 (or vica versa). Perhaps close #966 in favour of this one?

**Comment by elviskahoro** [comment]
There was an update today: v0.2022.04.25.09.59.stable_03

Can someone let us know if CMD+§ is working for people now?

**Comment by svekl** [comment]
Spent some time to set it up, it didn't work even in English first, hotkey was shown in preferences window, but window didn't appear. Turned off/on accessibility permissions, then switched languages randomly, then randomly reassigned hotkey few times and it started to work, not sure what exactly helped. And as @ideea mentioned - doesn't work in different language.

**Comment by stephen-murby** [comment]
CMD+§ Still isn't working for me after today's update, so I have just reverted to an alternate key binding. Will try again after a future update.

**Comment by pyronaur** [comment]
@elviskahoro Updated Warp today, it's still an issue with `CMD + §` - see #1156 for the exact settings/keyboard used.

**Comment by wutzi15** [comment]
Version v.0.2022.04.25.09.59.stable_05 still doesn't work for `ctrl + <` with a german keyboard layout.

**Comment by stephen-murby** [comment]
In the current version `v0.2022.04.25.09.59.stable_05` functionality seems to have regressed a little for me. I had previously opted for `F6` in the absence of being able to use `CMD + §`, but now that shortcut isn't working either.

**Comment by synchromation** [comment]
Yep, stopped working for me

**Comment by ephemer** [comment]
Reposting this here because the other issue seems less recent: `ctrl + backtick` is not working either (international english keyboard, late 2020 mbp)

<img width="503" alt="image" src="https://user-images.githubusercontent.com/5485935/166680572-afac0daa-a24f-43ed-8239-20bc563b95d5.png">

**Comment by kevinyang372** [comment]
Keybinding regression from last week's release should be fixed!

For the `CMD + §` issue, I suspect it's because Warp does not support overriding system keybindings yet. Check if you have a binding mapped to ``CMD + ` `` or ``CMD + §`` under `System Preferences > Keyboard > Shortcuts` and untick it -- does this fix for you?

For me the binding was
<img width="384" alt="image" src="https://user-images.githubusercontent.com/30107576/166830277-7ca32044-7693-4101-9bee-09c44a7525c0.png">

**Comment by wutzi15** [comment]
In v0.2022.05.02.09.00.stable_01 the keybinding on the german keyboard with `ctrl + <` works.

**Comment by pyronaur** [comment]
`v0.2022.05.02.09.00.stable_01` works for `cmd + §` for me.

**Comment by Froelund** [comment]
Just updated. Same for me ^
